app/javascript/employees.js

```diff
@@ -0,0 +1,17 @@
+ setActives = (employee) => {
+     var csrf = document.querySelector('meta[name="csrf-token"]').content;
+     var xhr = new XMLHttpRequest();
+     xhr.open("DELETE", "employees/"+ employee);
+     xhr.setRequestHeader('X-CSRF-Token', csrf);
+     xhr.send();
+     xhr.onload = function() {
+         if (xhr.status != 200) {
+             console.log('ERROR');
+         } else {
+             console.log('DELETED!');
+         }
+     };
+     xhr.onerror = function() {
+         console.log('NO CONNECTION');
+     };
```

Comment:
Check your indentation.

Remove the console.log(). Posting debug code on GitHub is not recommended.

Reply:
Fixed!